modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -89,13 +90,16 @@ describe('browser.tinymce.plugins.fullscreen.FullScreenPluginTest', () => {
         plugins: 'fullscreen link',
         base_url: '/project/tinymce/js/tinymce',
         setup: (editor: Editor) => {
-          lastEventArgs.set(null);
-          editor.on('FullscreenStateChanged', (e: Editor) => {
-            lastEventArgs.set(e);
+          firedEvents = [];
+          editor.on('FullscreenStateChanged ResizeEditor', (e: any) => {
+            firedEvents.push(e.type);
+            firedEvents.push(e.state);
           });
         }
       }, [ FullscreenPlugin, LinkPlugin ]);
 
+      afterEach(() => firedEvents = []);
```

Comment:
```suggestion
      afterEach(() => {
        firedEvents = [];
      });
```
This should never be inlined for before/after hooks as the return value is passed to the hook.

Reply:
Hmm... that's been missed in review then I guess sorry. They shouldn't be like that.